.github/workflows/Test-coverage.yaml

```diff
@@ -28,9 +29,8 @@ jobs:
 
       - name: Install Ubuntu dependencies
         run: |
-          sudo apt-get install libcurl4-openssl-dev
-          sudo apt-get install -y openmpi-bin
-          echo "CMDSTANR_RUN_MPI_TESTS=TRUE" >> $GITHUB_ENV
+          sudo apt-get install -y libcurl4-openssl-dev
+          sudo apt-get install -y openmpi-bin openmpi-common openmpi-doc libopenmpi-dev
```

Comment:
Test coverage linux runs on ubuntu-16.04 for now so this `Install Ubuntu dependencies` is a bit different then in the regular rcmdcheck. There are some other library issues with ubuntu-20.04 with `covr` on Github Actions (stringi package or something like that) so this will do for now.